src/create-index.ts

```diff
@@ -69,18 +69,19 @@ export type IndexType = {
     keyItemGroupMap: Map<Midashi, ItemGroup[]>;
     SudachiSynonymsItemGroup: Map<SudachiSynonyms, ItemGroup>;
 };
-let _ret: IndexType | null = null;
-export const createIndex = async (): Promise<IndexType> => {
-    if (_ret) {
-        return Promise.resolve(_ret);
+let _cache = new Map<boolean, IndexType>();
+export const createIndex = async ({ allowLexeme }: { allowLexeme: boolean }): Promise<IndexType> => {
+    if (_cache.has(allowLexeme)) {
```

Comment:
What is means the `_cache`?
`allowLexeme` is always same value during the linting.
I do not understant why use map for `_cache`.